crowbar_framework/app/models/service_object.rb

```diff
@@ -1300,8 +1300,9 @@ def apply_role(role, inst, in_queue, bootstrap = false)
 
       message = "Failed to apply the proposal to:\n"
       bad_nodes.each do |node|
+        pre_cached_nodes[node] ||= NodeObject.find_node_by_name(node)
         node_alias = pre_cached_nodes[node].alias
-        message += "#{node_alias}: #{node}\n"
+        message += "#{node_alias} (#{node}):\n"
```

Comment:
The colon should be in between like this "#{node_alias}: (#{node})\n" Having it at the end does not make much sense because there is no other message after that.